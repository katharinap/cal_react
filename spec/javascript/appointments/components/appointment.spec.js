import React from "react";
import Appointment from "appointments/components/appointment";
import renderer from "react-test-renderer";

test("Render appointment", () => {
  const appointment = {
    title: "My Appointment",
    appt_time: "2017-11-22 19:00"
  };
  const component = renderer.create(<Appointment appointment={appointment} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
