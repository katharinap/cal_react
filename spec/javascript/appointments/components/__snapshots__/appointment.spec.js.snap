// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render appointment 1`] = `
<div
  className="appointment"
>
  <h3>
    My Appointment
  </h3>
  <p>
    November 22 2017, 7:00 pm
  </p>
</div>
`;
